app.py

```diff
@@ -149,10 +149,14 @@ def user_entries(ack, respond, body, command, logger):
     try:
         user_id = body['user_id']
         name = full_name(user_id)
-        num_entries = int(command['text'])
+        argument = command['text']
+        if argument == "":
+            num_entries = 5
+        else:
+            num_entries = int(argument)
```

Comment:
```suggestion
num_entries = int(command['text'] if command['text'] != "" else 5
```
I think this looks nicer as a ternary